.test-infra/jenkins/job_PerformanceTests_Debezium.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as common
+import Kubernetes
+import InfluxDBCredentialsHelper
+import LoadTestsBuilder
+
+String jobName = "beam_PerformanceTests_Debezium"
+
+String kubernetesYmlPath = "src/.test-infra/kubernetes/postgres/postgres-service-for-debezium.yml"
+
+
+String task = ":sdks:python:apache_beam:testing:load_tests:run -PloadTest.mainClass=apache_beam.testing.load_tests.debezium_performance_test -Prunner=DirectRunner"
```

Comment:
Do we really want to be running this on the direct runner? That seems unusual?

Reply:
Also, any reason to pull this out into a variable? We only use it once and it seems like the convention is to inline it. Same question for the kubernetesYmlPath.

If we do keep it, we should fix the spacing.